docs/guides/got_scraping.mdx

```diff
@@ -49,35 +51,37 @@ async sendRequest(overrideOptions?: GotOptionsInit) => {
 
 By default, it's the URL of current task. However you can override this with a `string` or a `URL` instance if necessary.
 
-See the [Got documentation](https://github.com/sindresorhus/got/blob/main/documentation/2-options.md#url) for more details.
+*More details: [Got documentation](https://github.com/sindresorhus/got/blob/main/documentation/2-options.md#url).*
```

Comment:
What's the rationale behind this change? I don't mind, just asking.

Reply:
Originally @B4nan mentioned this `you` -> `we` thing on Slack, so basically I tried to change `[you] see more details` to something less direct/addressed 🤷‍♂️

But in the end, we're not doing it...

Reply:
😂 Ok I get it now :D That's kinda dumb because there's often `Read more` in other projects' docs.

Like I suggested, `you` refers explicitly to the developer reading the docs, so if *they* are given an option then the steps depend on *their* path *they* have chosen. So `you` would be appropriate. https://docs.oracle.com/en/java/

> Java SE lets you develop

There's an implied choice. You can choose Java or any other language.

OTOH `we` refers to both the developer and the project, there may be a thing that applies to both persons. For example a tutorial. https://docs.oracle.com/search/?q=we&category=java&product=en%2Fjava

> We will run the HelloWorld sample

Reply:
I think that's exactly what we're doing in the end 😄 But this PR was already merged before :/